lib/ansible/modules/packaging/os/yum_repository.py

```diff
@@ -273,19 +273,23 @@
     description:
       - Path to the directory containing the databases of the certificate
         authorities yum should use to verify SSL certificates.
+    aliases: [ ca_cert ]
```

Comment:
Why keeping the old names here, and adding the new ones only as an alias?

Reply:
the implementation hard-codes the internal option name to match the name in the yum repo file... it's a thin wrapper.  Full details in the commit message.